Admin/Infrastructure/Harvister/Network Creation/README.md

```diff
@@ -13,6 +18,9 @@ Click Create and fill out the information in the *Basics* window. If you chose t
 
 <img src="Images/VMC.png">
 
+> [!IMPORTANT]
+> Selecting the *Untagged* network is preferred to prevent any internal conflicts, and we have observed it is much more stable in our environment.
+
 In the *Route* window configure it to use the **Manual** mode so we can make this network *internal* and not able to immediately access the internet. Fill in a `CIDR` range that does not conflict with the host's network (In our case do not use addresses that conflict with `192.168.0.0/21`). You should also set the `Gateway` to be some address in the `CIDR` range you selected.
```

Comment:
I would add, how can someone determine and confirm the host's network CIDR range?
For the purpose of longevity of this document, in the event it ever changes.